scripts/npm/prepublish/lib/docs.js

```diff
@@ -0,0 +1,46 @@
+var exec = require('child-process-promise').exec;
+var util = require('../../lib');
+
+function docs(version) {
+  checkoutDocs()
+  .then(updateDeps)
+  .then(build)
+  .then(commit)
+  .then(push)
+  .then(checkoutMaster)
+  .then(finish);
+}
+
+function checkoutDocs() {
+  util.printLn.info('Checking out gh-pages branch...');
+  return util.git.checkoutDocs();
+}
+
+function updateDeps() {
+  util.printLn.info('Updating the dependencies');
+  return exec('npm update');
+}
+
+function build() {
+  util.printLn.info('Rebuilding the project');
+  return exec('npm run build');
+}
+
+function commit(version) {
+  util.printLn.info('Committing dependency changes');
+  return util.git.commit('Rebuilding the docs for Capital Framework ' + version);
+}
+
+function push() {
+  return exec('git push gh-pages origin');
```

Comment:
Switch this out with `return exec('git push "https://' + process.env.GH_TOKEN + '@github.com/cfpb/capital-framework.git" gh-pages')`